pkg/policy/api/ingress.go

```diff
@@ -108,12 +108,35 @@ type IngressRule struct {
 	//
 	// +optional
 	FromEntities EntitySlice `json:"fromEntities,omitempty"`
+
+	aggregatedSelectors EndpointSelectorSlice
```

Comment:
Same here for the mix of API vs implementation details.

Reply:
Same.